Sources/_StringProcessing/Regex/Options.swift

```diff
@@ -161,8 +159,47 @@ extension RegexComponent {
   }
 }
 
+/// A built-in regex character class kind.
+///
+/// Pass one or more `RegexCharacterClassKind` classes to `asciiOnlyClasses(_:)`
+/// to control whether character classes match any character or only members
+/// of the ASCII character set.
 @available(SwiftStdlib 5.7, *)
+public struct RegexCharacterClassKind: OptionSet, Hashable {
```

Comment:
Did you consider or debate whether `RegexBuilder.CharacterClass` should be `Swift.RegexCharacterClass`? Then this would be a `Kind` under it.